### DISCUSSION TITLE
Is there any plan to release a new version?

### DISCUSSION THREAD

**Discussion by adrianoc** [jbevain/cecil#883] [General]
Hey, I wonder if there's an eta or some schedule for the next official Cecil release.  I am implementing support for generic attributes in Cecilifier and that requires fixes not included in 0.11.4

**Comment by jbevain** [comment]
Hey @adrianoc yeah I want to publish a 0.11.5 early this year :)

**Reply by adrianoc** [reply]
thanks, that is amazing
